src/cryptography/hazmat/primitives/serialization/ssh.py

```diff
@@ -613,11 +627,21 @@ def serialize_ssh_private_key(private_key, password=None):
         ciph.encryptor().update_into(buf[ofs:mlen], buf[ofs:])
 
     txt = _ssh_pem_encode(buf[:mlen])
-    buf[ofs:mlen] = bytearray(slen)
+    # Ignore the following type because mypy wants
+    # Sequence[bytes] but what we're passing is fine.
```

Comment:
Err, mypy doesn't consider `bytearray` to be a `Sequence[bytes]`? Can we please file a bug with them on this.

Reply:
Correct:

```python
buf = memoryview(bytearray(b"a" * 500))
buf[3:4] = bytearray(1)
```
```
test.py:2: error: Incompatible types in assignment (expression has type "bytearray", target has type "Sequence[bytes]")
```

Reply:
Please file a mypy bug on this.

I suspect you can work around this with `slen_buf: Sequence[bytes] = bytearray(slen); buf[ofs:mlen] = slen_buf` though.

Reply:
That doesn't work

```
error: Incompatible types in assignment (expression has type "bytearray", variable has type "Sequence[bytes]")
```

Reply:
Welp. Anyways, mypy bug please.